docker_templates/templates/snippet/install_ros_bootstrap_tools.Dockerfile.em

```diff
@@ -1,20 +1,25 @@
 @{
-if int(ros_version) == 2:
-    package_list = [
-        'build-essential',
+import rosdistro
+index = rosdistro.get_index(rosdistro.get_index_url())
+dist_info = index.distributions[ros_distro]
+ros_version = int(dist_info['distribution_type'][-1])
+ros_python_version = dist_info['python_version']
+prefix = 'python'
+if ros_python_version == 3:
+    prefix += str(ros_python_version)
+package_list = [
+    'build-essential',
+    f'{prefix}-rosdep',
+]
+if ros_version == 2:
+    package_list += [
         'git',
-        'python3-colcon-common-extensions',
-        'python3-colcon-mixin',
-        'python3-rosdep',
```

Comment:
What happened to `-rosdep` if the `ros_version` is 2?

Reply:
rosdep and build-essential are installed regardless of the ROS version (L11-12 of this file)